cobalt/script/v8c/v8c_global_environment.cc

```diff
@@ -394,24 +391,16 @@ void V8cGlobalEnvironment::MessageHandler(v8::Local<v8::Message> message,
   v8::Isolate* isolate = v8::Isolate::GetCurrent();
   V8cGlobalEnvironment* global_environment =
       V8cGlobalEnvironment::GetFromIsolate(isolate);
-  if (isolate->GetEnteredContext().IsEmpty()) {
-    return;
-  }
```

Comment:
I think this is to support some testing scenarios where there is no global object.

Reply:
And for this:

> Use GetEnteredOrMicrotaskContext().